docs/solana/output.rst

```diff
@@ -1,9 +1,279 @@
+
+.. container:: call-trace-analysis
+   :name: call-trace-example
+      
+   .. image:: img/call_trace_detail.png
+        :alt: A call trace example
+        :align: right
+        :width: 300px
+
+      Call trace output showing counterexample values
+
+   Let's analyze this counterexample:
+
+   1. **Nondet Values**: 
+
+        * The prover chose ``101`` for ``amount1`` and implicitly ``2`` for ``amount2``
+        * These values are chosen to demonstrate the violation of our assertions
+        * ``<?>`` means that the value is not important for the counterexample 
+
+   2. **Variable States**:
+
+        * ``is_great`` is assumed, thus ``1`` (true) as ``amount1`` (101) is indeed greater than 100
+        * ``amount1`` is 101, which satisfies our assumption ``cvt_assume!(is_great)``
+        * ``is_great2`` is ``0`` (false) because ``amount2`` (2) is not greater than 100
+        * ``amount2`` is 2, which satisfies our assumption ``cvt_assume!(amount2 != 10)``
+
+   3. **Assertion Failure**:
+
+        * The assertion ``cvt_assert!(amount1 < 100)`` fails because we have a contradiction:
+            * We assumed ``is_great`` is true, which means ``amount1 > 100``
+            * But then we assert ``amount1 < 100``
+        * This is impossible to satisfy, hence the counterexample
+
+
+Rule Sanity Checking
+------------------
+
+To ensure rules aren't passing vacuously (due to contradictory assumptions), add sanity checking to your configuration:
+
+.. code-block:: json
+
+    {
+        "rule_sanity": "basic"
+    }
+
+This adds an implicit ``cvt_assert!(false)`` at the end of each rule. If this assertion is unreachable, it confirms that:
+
+1. Your assumptions aren't contradictory
+2. The rule's success is meaningful
+
+Common Sanity Check Results
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. **Sanity Check PASSED**: Desired outcome - confirms rule isn't vacuously true
+2. **Sanity Check WARNING**: Warning sign - indicates contradictory assumptions
+
+See `Sanity <#solana-sanity-section>`_ for more details.
+
+
+Best Practices
+------------
+
+1. Always enable rule sanity checking in your configuration
+2. Use ``nice_print!`` macro for clearer counterexample analysis
+3. Review full call traces when investigating failures
+4. Validate counterexamples against your program's expected state space
+
+Advanced Topics
+-------------
+
+Over-approximation Detection
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When analyzing counterexamples, consider:
+
+1. Initial state feasibility: Is the starting state a reachable state for the code you're analyzing?
+2. Transaction sequence validity: Does the computation path make sense, did you overlook a certain scenario?
+3. State transition legitimacy: Do the values throughout the computation match? Are the parameters to your rule changing as expected?
+
+If any seem impossible in your actual program, the counterexample might be due to over-approximation.
+
+
+The Dump
```

Comment:
Generally @urikirsh says we shouldn't talk about TAC dumps to external users. For Solana, this however is the only way to inspect the results. I suggest we use the term TAC dump and not just dump here.

```suggestion
The TAC Dump
```